core/chains/evm/log/pool.go

```diff
@@ -50,14 +52,16 @@ type logPool struct {
 	// This min-heap maintains block numbers of logs in the pool.
 	// it helps us easily determine the minimum log block number
 	// in the pool (while the set of log block numbers is dynamically changing).
-	heap *pairingHeap.PairHeap
+	heap   *pairingHeap.PairHeap
+	logger logger.Logger
 }
 
-func newLogPool() *logPool {
+func newLogPool(logger logger.Logger) *logPool {
```

Comment:
Please call the variable name `lggr`, for consistency with other places in the code.